dev/com.ibm.ws.session.store/build.xml

```diff
@@ -0,0 +1,18 @@
+<!--
```

Comment:
this file can be removed

Reply:
Thanks for catching these.  I removed both.